wa/Chart.yaml

```diff
@@ -47,7 +47,7 @@ dependencies:
     repository: 'https://hmctspublic.azurecr.io/helm/v1/repo/'
     condition: wa.camunda.enabled
   - name: postgresql
-    version: 1.0.0
+    version: 1.0.2
```

Comment:
Did you mean to bump 2 versions or 1?  Think your PR used 1.0.1-beta?  https://github.com/hmcts/et-ccd-definitions-admin/pull/196/files#diff-ee1c69f4a5054b5a43bcf3707ab0c364d841b4f0b387e5c877884b1eaec177ceR51